docs/modules/ROOT/pages/how-tos/testing/integration/periodic-integration-tests.adoc

```diff
@@ -0,0 +1,134 @@
+= Triggering Periodic Integration Tests
+
+In {ProductName}, you can run integration tests on a schedule using Kubernetes CronJobs.
+
+== Procedure
+
+To trigger a periodic Integration Test, complete the following steps:
+
+. Create an integration test by following the xref:how-tos/testing/integration/creating.adoc[Creating a integration test guide].
+
+[NOTE]
+====
+If the Integration Test is designed to run periodically rather than during Pull Request checks or post-merge, you need to add the *disabled* xref:how-tos/testing/integration/choosing-contexts.adoc[context] to the Integration Test.
+====
+
+. Create RBAC rules in your tenant to allow CronJob ServiceAccount to access snapshot.
+
+. Create a link:https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/[CronJob] that will trigger periodic Integration Tests via xref:how-tos/testing/integration/rerunning.adoc[snapshot labels].
+
+=== Procedure with example
+
+This procedure assumes that you already have an integration test created and a ServiceAccount available in your tenant.
+
+. Create a `Role` and `RoleBinding` to grant the necessary snapshot access to the ServiceAccount, which will be used to trigger the periodic Integration Tests
+[source,yaml]
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: periodic-jobs-role
+  namespace: default-tenant
+rules:
+  - verbs:
+      - get
+      - watch
+      - list
+      - update
+      - patch
+    apiGroups:
+      - appstudio.redhat.com
+    resources:
+      - snapshots
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: periodic-jobs-rolebinding
+  namespace: default-tenant
+subjects:
+  - kind: ServiceAccount
+    name: default-tenant
+    namespace:
+roleRef:
+  kind: Role
+  name: periodic-jobs-role
+  apiGroup: rbac.authorization.k8s.io
```

Comment:
I think you are missing the dashes around this to group it as the source.

Reply:
@arewm If I put dashes the sentence `. Create a CronJob that will trigger a Periodic via snapshot labels.` will appear like 1. Create a Cr... instead of 2. Create a CronJob...